c2corg_ui/templates/index.html

```diff
@@ -37,14 +37,29 @@ <h1 class="welcome-title text-center" translate>The mountain sports community</h
 
     </div>
   </section>
-  <section ng-class="{'show': userCtrl.auth.isAuthenticated()}" class="site-intro">
-    <div class="intro-banner">
-      <h1 class="welcome-title text-center" translate>Activity feed</h1>
-    </div>
-  </section>
 
   <div class="feed">
-    <button class="btn orange-btn" id="add-my-outing" protected-url-btn url="${request.route_path('outings_add')}"><span class="glyphicon glyphicon-plus-sign"></span><span translate>add an outing</span></button>
-    <app-advanced-search document-type="outings" use-map="false"></app-advanced-search>
+    <div app-feed location="home">
+      <div class="feed-list" ng-show="feedCtrl.documents.length > 0">
+
+        <article class="feed-list-item" protected-url-btn url="${request.route_path('outings_add')}">
+          <div class="list-item vertical-align">
+            <button class="orange-btn btn">add an outing</button>
+          </div>
+        </article>
+
+        <article class="feed-list-item" ng-repeat="doc in feedCtrl.documents"
+                 ng-init="type = feedCtrl.documentType(doc.document.type);">
+          <p class="action" ng-cloak><b><a ng-href="todo">{{doc['user']['username']}}</a></b>
+            <span x-translate>has {{doc.change_type}}</span> a(n)
```

Comment:
For outings it would also be cool to show something like "xyz went hiking with ...".